src/Signer/None.php

```diff
@@ -1,26 +0,0 @@
-<?php
-declare(strict_types=1);
```

Comment:
We should probably mention somewhere (in the docs?) that we deviate from the spec here

Reply:
Good catch, commit is coming